application/src/main/java/com/sanction/thunder/authentication/ThunderAuthenticator.java

```diff
@@ -1,32 +1,43 @@
 package com.sanction.thunder.authentication;
 
 import com.google.common.base.Optional;
-
+import com.sanction.thunder.dao.PilotUsersDao;
+import com.sanction.thunder.models.PilotUser;
 import io.dropwizard.auth.AuthenticationException;
 import io.dropwizard.auth.Authenticator;
-
 import io.dropwizard.auth.basic.BasicCredentials;
 
-import java.util.List;
+import javax.inject.Inject;
+
+import static org.eclipse.jetty.http.HttpParser.LOG;
```

Comment:
This is not the right way to use the logger. You to remove this import and then you need a class field like this:

```java
private static final Logger LOG = LoggerFactory.getLogger(ThunderAuthenticator.class);
```
Then you can use the `LOG` like normal.